.gitignore

```diff
@@ -32,6 +32,7 @@ web/.htaccess
 .env.*
 !.env.example
 !.env.pantheon
+!.env.local-sample
```

Comment:
is there a difference between `.env.example` and `.env.local-sample`? Is there a specific reason for both?
If not, I would just pick one and use that.

Reply:
Great point, I'll adjust!